### ISSUE TITLE
How can I change the pygments 'style' in decompilation/diff view?

### ISSUE THREAD

**Issue by agnosticlines** [joxeankoret/diaphora#312]
Hi Joxean, I've written a pygments style file for my theme of choice and I'd like to hook it up to diaphora, how can I go about doing this? There doesn't seem to be a way to modify it for example in the cfg file, diaphora is a fantastic tool but I always have to change my IDA theme back to the default one to get the diff/patch view working properly :-(

**Comment by joxeankoret** [comment]
I don't really know, to be honest. Let me take a look to it tonight...

**Comment by agnosticlines** [comment]
Fair enough! I think I've offered to sponsor full theme support for diaphora before, it's the one obstacle I have to using it, I really really love my custom theme, I'd love it if it just took IDA theme support, but not sure if there's a 1->1 mapping for those and diaphora's UI elements, like the diff colours etc

**Comment by joxeankoret** [comment]
Uhm... are you using the official IDA themes support or a plugin? If it's the official method, I believe I could somehow easily support it.

**Comment by agnosticlines** [comment]
Yeah official ida themes with CSS https://github.com/catppuccin/ida-debugger etc

**Comment by joxeankoret** [comment]
Note to myself: it can be implemented following the recommendation of Igor to read from the registry the current theme, and then parsing the CSS of the theme to get the colours. Not terribly trivial, but can be implemented.

**Comment by agnosticlines** [comment]
> Note to myself: it can be implemented following the recommendation of Igor to read from the registry the current theme, and then parsing the CSS of the theme to get the colours. Not terribly trivial, but can be implemented.

Happy to donate to some goal of getting it landed if that'd help

**Comment by joxeankoret** [comment]
Sorry for taking years... there is a patch integrated today by a 3rd person using themes that seems to fix this.
